src/GovUk.Education.ExploreEducationStatistics.Content.Model/Extensions/ReleaseAmendmentExtensions.cs

```diff
@@ -174,23 +187,23 @@ private static void UpdateAmendmentContent(Release.CloneContext context)
             );
         }
 
-        public static void CreateGenericContentFromTemplate(this Release release, Release newRelease)
+        public static void CreateGenericContentFromTemplate(
+            this Release originalRelease, 
+            Release.CloneContext context)
         {
-            var context = new Release.CloneContext(newRelease);
-
-            newRelease.Content = release.Content
-                .Where(c => c.ContentSection.Type == ContentSectionType.Generic)
+            var newRelease = context.NewRelease;
+            
+            newRelease.Content = originalRelease.Content
+                .Where(section => section.Type == ContentSectionType.Generic)
                 .ToList();
 
-            newRelease.Content = newRelease
+            newRelease.Content = originalRelease
                 .Content
-                .Select(rcs => rcs.Clone(context))
+                .Select(section => section.Clone(context))
                 .ToList();
 
-            newRelease.ContentBlocks = newRelease
-                .ContentBlocks
-                .Select(rcb => rcb.Clone(context))
-                .ToList();
+            // TODO EES-4467 - we can add the cloning of Data Blocks back in here rather than being doing separately
+            // when the new DataBlock / DataBlockVersion tables are added.
```

Comment:
So this isn't used for creating new Releases themselves (i.e. creating a "2023" Release when you've already got a "2022" Release).  This is for making *Amendments* to your "2022" Release.

Reply:
In Rider, I see `ReleaseAmendmentExtensions#CreateGenericContentFromTemplate` which is used once by `ReleaseService#CreateGenericContentFromTemplate` and that in turn is used once by `ReleaseService#CreateRelease`, which I believe is for creating a release for a new time period.

If that's right, not sure why the method is in `ReleaseAmendmentExtensions`.

Reply:
Ahh yes I see what you mean!  Well I'll remove the comments from here, and also move this whole method elsewhere cos as you say, it's in the wrong place!

Also, see the deliberate mistake in the original code that meant that it wasn't doing anything at all!

```
            newRelease.ContentBlocks = newRelease
                .ContentBlocks
                .Select(rcb => rcb.Clone(context))
                .ToList();
```

Reply:
Gonna have to go back to dev and try this out to see what happens to the new Release if the previous one had Data Blocks :thinking:

Reply:
So it turns out we were getting away with it because we were never initialising `Release.ContentBlocks` in the EF query:

```
            var templateRelease = await _context.Releases.AsNoTracking()
                .Include(r => r.Content)
                .ThenInclude(c => c.ContentSection)
                .FirstAsync(r => r.Id == releaseId);
```

So this code was wrong but harmless, and is safe to remove!